### PR TITLE
Adding Kubos core services packages

### PR DESCRIPTION
**Pull request by CatGarab**
Adding KLB packages for the core services (core services = critical Kubos services whose functions are specifically for the main OBC, not any particular payload hardware)

They are gathered under a singular package option to:
a) make it easy to include all of them
b) make it an all-or-nothing type of thing

Packages currently included:
- Telemetry DB
- All the Lua services (which will be broken out into their own packages at some point)
- The application service - kubos/kubos#91

### CODE REVIEW

**Comment by CatGarab** [comment]
[Kubos Core KLB packages](https://trello.com/c/IOQ9kA5B/342-kubos-core-klb-packages)

File: package/kubos/kubos-core/Config.in [hunk 1]
```diff
@@ -0,0 +1,15 @@
+menuconfig BR2_PACKAGE_KUBOS_CORE
+   bool "Kubos Core Services"
+   default n
+   select BR2_PACKAGE_KUBOS_CORE_APP_SERVICE
+   select BR2_PACKAGE_KUBOS_CORE_LUA
+   select BR2_PACKAGE_KUBOS_CORE_TELEMETRY_DB
+   help
+       Include the KubOS core services. 
+       This includes things like the telemetry database service, shell service, and file transfer service.
```
Comment (jacoffey3) [inline package/kubos/kubos-core/Config.in]: Remove "things like". Just list what it has.

File: package/kubos/kubos-core/Config.in [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+       If you are building a full KubOS image, this option MUST BE SELECTED.
```
Comment (jacoffey3) [inline package/kubos/kubos-core/Config.in]: Not necessarily true. If you want to just use Kubos Linux, and not our middleware you don't need it.

File: package/kubos/kubos-core/kubos-core-telemetry-db/kubos-core-telemetry-db.mk [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#
+###############################################
+
+KUBOS_CORE_TELEMETRY_DB_POST_BUILD_HOOKS += TELEMETRY_DB_BUILD_CMDS
+KUBOS_CORE_TELEMETRY_DB_POST_INSTALL_TARGET_HOOKS += TELEMETRY_DB_INSTALL_TARGET_CMDS
+KUBOS_CORE_TELEMETRY_DB_POST_INSTALL_TARGET_HOOKS += TELEMETRY_DB_INSTALL_INIT_SYSV
+
+define TELEMETRY_DB_BUILD_CMDS
+	cd $(BUILD_DIR)/kubos-$(KUBOS_VERSION)/services/telemetry-service && \
+	PATH=$(PATH):~/.cargo/bin:/usr/bin/iobc_toolchain/usr/bin && \
```
Comment (plauche) [inline package/kubos/kubos-core/kubos-core-telemetry-db/kubos-core-telemetry-db.mk]: Why do we default to adding the iobc's toolchain to the path?

**Pull request merged**